libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -676,6 +672,15 @@ uint32_t AP_SerialManager::find_baudrate(enum SerialProtocol protocol, uint8_t i
     return _state->baudrate();
 }
 
+void AP_SerialManager::set_default_baud(enum SerialProtocol protocol, uint8_t instance, uint32_t _baud)
```

Comment:
I think this should be `set_and_default_baud`. That is exactly the AP Param call. It has both `set_default` which won't change the current value and `set_and_default` which does.

Reply:
Yep, fixed.